apps/web/src/config/abi/permit2.json

```diff
@@ -0,0 +1,42 @@
+[
```

Comment:
let's move the abi to our permit2 sdk package. And let's use ts file to export the abi for better typing